problems/arraysIntro.js

```diff
@@ -4,57 +4,77 @@
  * @param {string} fruit
  * @returns {string[]}
  */
-function addFruitToEnd() {}
+function addFruitToEnd(fruits, fruit) {
+  fruits.push(fruit)
+  return fruits
+  
+}
+
 
 /**
  * Adds a fruit to the front of an array of fruits and returns the array.
  * @param {string[]} fruits
  * @param {string} fruit
  * @returns {string[]}
  */
-function addFruitToFront() {}
+function addFruitToFront(fruits, fruit) {
+  fruits.unshift (fruit)
+  return fruits
+}
 
 /**
  * Removes the last fruit from an array of fruits and returns the array.
  * @param {string[]} fruits
  * @returns {string[]}
  */
-function removeLast() {}
+function removeLast(fruits) {
+  fruits.pop()
+  return fruits
+}
 
 /**
  * Removes and returns the last fruit from from an array of fruits
  * @param {string[]} fruits
  * @returns {string} The fruit that was removed
  */
-function removeLastandReturnLast() {}
+function removeLastandReturnLast(fruits) {
+return fruits.pop()
```

Comment:
Remember to tab in lines that are inside of a function. Use VS Code Prettier option + shift + f to format your code.